modules/base.py

```diff
@@ -194,6 +195,17 @@ class K8sServiceModuleProcessor(ModuleProcessor):
     # TODO(patrick): Remove this flag and references to it once all clouds support multiple ports
     FLAG_MULTIPLE_PORTS_SUPPORTED = False
 
+    def pre_hook(self, module_idx: int) -> None:
+        pending_upgrade_helm_chart = Helm.get_helm_list(
+            release=f"{self.layer.name}-{self.module.name}"
+        )  # , status="pending-upgrade")
```

Comment:
The status check is disabled here which does not appear to be intended behavior, since that would cause the below `if` statement to always be `True` if the chart exists, even if installed correctly.